contracts/.env.example

```diff
@@ -1,18 +1,27 @@
 # RPC_URL is the URL of the Ethereum node to use for deployment
-BASE_ANVIL_RPC_URL="http://127.0.0.1:8545"
+BASE_SEPOLIA_RPC_URL=
+BASE_ANVIL_RPC_URL="${BASE_ANVIL_RPC_URL:-http://127.0.0.1:8545}"
+RIVER_ANVIL_RPC_URL="${RIVER_ANVIL_RPC_URL:-http://127.0.0.1:8546}"
 
 # PRIVATE_KEY is the private key of the account to use for deployment
 LOCAL_PRIVATE_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
 PRODUCTION_ADDRESS=0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266
+SENDER_ADDRESS=
+HD_PATH="m/44\'/60\'/0\'/0/0"
```

Comment:
This derivation path is not accurate for all hardware wallets so I don't think we should include this.

Reply:
I leave it as a common example. It is also [suggested by Ledger](https://www.ledger.com/blog/understanding-crypto-addresses-and-derivation-paths).